hm11_2.py

```diff
@@ -0,0 +1,36 @@
+""" Task 2: bank deposit """
+
+
+class Bank:
+
+    """ h """
+    persent = 30
```

Comment:
where are you using this class attribute?

Reply:
It is the Pylinter wishes :(
hm11_2.py:17:8: W0201: Attribute 'percent' defined outside init (attribute-defined-outside-init)

Reply:
why you can not move it to `__init__`?
```
    def __init__(self, summ_dep, dep_time, dep_n, persent=0.1):
        self.summ_dep = summ_dep
        self.dep_time = dep_time * 12
        self.dep_n = dep_n
        self.persent = persent
```